fivox/compartmentLoader.cpp

```diff
@@ -49,8 +49,6 @@ class CompartmentLoader::Impl
         , _report( _config.getReportSource( params.getReport( )),
                    brion::MODE_READ, _target)
     {
-        _report.updateMapping( _target );
-
```

Comment:
So updateMapping can be protected!? If no, doxygen of updateMapping needs an explanation.

(not for this PR, but I'm wondering what the point then is)